### PR TITLE
Fix world map configuration typos

### LINKED ISSUE #3298: World map configuration typos
The spellbook options have "Destionation" instead of "destination".  
Also, I'd capitalise the spellbook type, at least the Arceeus one, but that's just personal preference.

### PR DESCRIPTION
**Pull request by shmeeps**
Fixes #3298

### CODE REVIEW

**Comment by deathbeam** [comment]
I would rename spellbooks to Standard and Lunar (e.g capitalized too) otherwise it looks weird in config listing.

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ default boolean agilityShortcutLevelIcon()
 
 	@ConfigItem(
 		keyName = WorldMapPlugin.CONFIG_KEY_NORMAL_TELEPORT_ICON,
-		name = "Show standard spellbook destionations",
+		name = "Show standard spellbook destinations",
```
Comment (MESLewis) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Standard

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java [hunk 2]
```diff
@@ -93,7 +93,7 @@ default boolean normalTeleportIcon()
 
 	@ConfigItem(
 		keyName = WorldMapPlugin.CONFIG_KEY_ANCIENT_TELEPORT_ICON,
-		name = "Show ancient magicks destionations",
+		name = "Show ancient magicks destinations",
```
Comment (MESLewis) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Ancient Magicks

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java [hunk 3]
```diff
@@ -104,7 +104,7 @@ default boolean ancientTeleportIcon()
 
 	@ConfigItem(
 		keyName = WorldMapPlugin.CONFIG_KEY_LUNAR_TELEPORT_ICON,
-		name = "Show lunar spellbook destionations",
+		name = "Show lunar spellbook destinations",
```
Comment (MESLewis) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Lunar

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java (hunk 2 above)
Comment (MESLewis) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Now I'm conflicted on if Magicks should be capitalized.

Comment (JeljoGb) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: I would capitalise it. It's a title of a thing.

Comment (MESLewis) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Spellbook could also be capitalized too. I'm not sure how it is actually done in game or if it's even consistent in game. Basically that's why I originally just made it all lower case, it's easier :P

Comment (JeljoGb) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Yeah, I can understand that, but I've long stopped looking at how Jagex does things for consistency.

Comment (shmeeps) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java]: Wiki has them all as standard spellbook, Lunar spellbook, Arceuus spellbook and Ancient Magicks. I'm fine with capitalizing it all though.

**Comment by shmeeps** [comment]
Updated

**Pull request merged**